packages/react-output-target/src/react-output-target/create-component-wrappers.ts

```diff
@@ -9,18 +11,18 @@ export const createComponentWrappers = async ({
   outDir,
   customElementsDir,
   esModules,
-  experimentalUseClient,
   excludeComponents,
   project,
+  hydrateModule,
 }: {
   stencilPackageName: string;
   components: ComponentCompilerMeta[];
   customElementsDir: string;
   outDir: string;
   esModules?: boolean;
-  experimentalUseClient?: boolean;
```

Comment:
Will older environments be able to parse (or skip) the `use client;` directive annotation in JS files?

Reply:
Ahh I see, the `use client;` was moved to only be generated as part of the Next.js component wrappers. The config option was replaced with the hydrate module. 

This makes sense - I don't have concerns 👍